### PR TITLE
Fix homebridge logger import that prevents the plugin from loading

### PR DESCRIPTION
**Pull request by poblouin**

### CODE REVIEW

File: src/web/api/http-server.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { Logging } from 'homebridge';
 import serveStatic from 'serve-static';
 import { NextFunction, Request, Response } from 'express-serve-static-core';
 import * as url from 'url';
-import { withPrefix } from 'homebridge/lib/logger';
```
Comment (poblouin) [inline src/web/api/http-server.ts:14]: This is deprecated and the plugin doesn't load

Comment (madchicken) [reply src/web/api/http-server.ts:14]: Wait, it works here. The deprecation should not affect loading at all. What is your error?

Comment (poblouin) [reply src/web/api/http-server.ts:14]: This is what I got:

```
ERROR LOADING PLUGIN homebridge-zigbee-nt:
[2/12/2021, 12:24:59 PM] Error: Cannot find module 'homebridge/lib/logger'
Require stack:
- /homebridge/node_modules/homebridge-zigbee-nt/dist/web/api/http-server.js
- /homebridge/node_modules/homebridge-zigbee-nt/dist/platform.js
- /homebridge/node_modules/homebridge-zigbee-nt/dist/index.js
- /usr/local/lib/node_modules/homebridge/lib/plugin.js
- /usr/local/lib/node_modules/homebridge/lib/pluginManager.js
- /usr/local/lib/node_modules/homebridge/lib/api.js
- /usr/local/lib/node_modules/homebridge/lib/server.js
- /usr/local/lib/node_modules/homebridge/lib/cli.js
- /usr/local/lib/node_modules/homebridge/bin/homebridge
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/homebridge/node_modules/homebridge-zigbee-nt/src/web/api/http-server.ts:14:1)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
```

Using your plugin v2.0.5
homebridge v1.2.5
homebridge-config-ui-x@4.38.0

Comment (poblouin) [reply src/web/api/http-server.ts:14]: I dinged a bit more, the problem is that this library relies on hb 1.1.6 and it is not compatible with the version used by my docker image (1.2.5). With v1.1.7 the logger works fine, but the WebSocket server is broken and no devices are discovered.. I'll check a bit more, but I might just downgrade

**Review by madchicken — COMMENTED** [review]
Your version does not compile because they messed up with exported types in Homebridge.
If you do a `yarn build` you get this:
```
src/web/api/http-server.ts:89:16 - error TS1362: 'Logger' cannot be used as a value because it was exported using 'export type'.

89     this.log = Logger.withPrefix('ZigBee');
                  ~~~~~~

  node_modules/homebridge/lib/index.d.ts:37:15
    37 export type { Logger, Logging, } from "./logger";
                     ~~~~~~
    'Logger' was exported here.


Found 1 error.
```

File: src/web/api/http-server.ts (hunk 1 above)
Comment (madchicken) [reply src/web/api/http-server.ts:14]: oh, that's interesting. Let me try to upgrade hb then

Comment (madchicken) [reply src/web/api/http-server.ts:14]: could you try using the `unstable` plugin tag? Version 2.1.0-unstable.3 could fix the issue for you

Comment (poblouin) [reply src/web/api/http-server.ts:14]: I tried and it doesn't unfortunately :(

**Comment by poblouin** [comment]
> Your version does not compile because they messed up with exported types in Homebridge.
> If you do a `yarn build` you get this:
> 
> ```
> src/web/api/http-server.ts:89:16 - error TS1362: 'Logger' cannot be used as a value because it was exported using 'export type'.
> 
> 89     this.log = Logger.withPrefix('ZigBee');
>                   ~~~~~~
> 
>   node_modules/homebridge/lib/index.d.ts:37:15
>     37 export type { Logger, Logging, } from "./logger";
>                      ~~~~~~
>     'Logger' was exported here.
> 
> 
> Found 1 error.
> ```

Oh I thought I tried the build, that must have been with another version. Sorry I did multiple things at the same time yesterday to make it work 😅

I ended up installing `homebridge-zigbee-nt v2.0.1` and it works. I won't upgrade for a while I guess for now. If I have some spare time soon, I will test with another docker image, that is not my main server, because it took me few hours yesterday to make it work again haha

File: src/web/api/http-server.ts (hunk 1 above)
Comment (poblouin) [reply src/web/api/http-server.ts:14]: Added a comment below, following this, we can close this PR for now

**Pull request closed without merging**